packages/iov-rise/src/derivation.ts

```diff
@@ -25,3 +25,20 @@ export function pubkeyToAddress(pubkey: Uint8Array): Address {
   const addressString = Long.fromBytesLE(firstEightBytes, true).toString(10) + "R";
   return addressString as Address;
 }
+
+export function isValidAddress(address: string): boolean {
+  return isValidAddressWithEnding(address, "R");
+}
+
+// copied from iov-lisk, maybe just import this instead?
+export function isValidAddressWithEnding(address: string, ending: string): boolean {
```

Comment:
Nice duplication, like it. It is a perfect first step to extract into a shared dpos package. I'd avoid adding cross-dependencies between the chains.